File: submissions/sm_026_rohit-kumar/week_03/day_6/assignments/pyramidPlay.js
function pyramid(level) {
    for(var i = 0; i < level; i++) {
        str = ''
        for(var j = 0; j < level-i; j++)
            str = str + '.'
        
        str = str + '0'
        for(var k = 0; k < i; k++){
            str = str + '.0'
        }
        for(var j = 0; j < level-i; j++)
            str = str + '.'

        console.log(str)
    }   
}


function inversePyramid(level) {
    for(var i = 0; i < level; i++) {
        str = ''
        for(var j = 0 ; j <= i; j++) {
            str = str + '.'
        }
        str = str +'0'
        for(var k = 0; k < level-1 - i; k++) {
            str = str + '.0'
        }
        for(var j = 0 ; j <= i; j++) {
            str = str + '.'
        }

        console.log(str)
    }
}


function rhombus(level) {
    pyramid(level)
    inversePyramid(level)
}



var level = 6
rhombus(level)

File: submissions/sm_103_apoorva/week_13/day_5/pyramid_play.py
level = int(input("Enter the level of pyramid: "))

def pyramid_list(level):
    li = []
    string = ""
    for i in range(level):
        for j in  range(level-i):
            string += "."
        for j in range(i+1):
            string += "0."
        for j in range(level-i-1):
            string += "."
        li.append(string)
        string = ""
    return li

def pyramid(level):
    li = pyramid_list(level)
    for i in li:
        print(i)

def inverse_pyramid(level):
    li = pyramid_list(level)
    li.reverse()
    for i in li:
        print(i)

def rhombus(level):
    li = pyramid_list(level)
    for i in li:
        print(i)
    li.reverse()
    reverse = li[1:]
    for i in reverse:
        print(i)

pyramid(level)
print("\n")
inverse_pyramid(level)
print("\n")
rhombus(level)


